Lesson_4_json_xml/Data.xml

```diff
@@ -6,7 +6,7 @@
         <qualification>"trainee"</qualification>
         <experience> false </experience>
         <practice> null </practice>
```

Comment:
XML does not support null concept.
To define an element with undefined value use empty tag <practice></practice>